tutorials/02_NeMo_Adapters.ipynb

```diff
@@ -1657,9 +1657,7 @@
         "id": "iz2wF3cd-6MF"
       },
       "outputs": [],
-      "source": [
-        ""
```

Comment:
could you revert any changes for irrelevant files, such as Jupiter notebooks like this? There are so many changes not expected.